cranelift/codegen/src/machinst/vcode.rs

```diff
@@ -673,8 +673,9 @@ impl<I: VCodeInst> VCode<I> {
         self.insts.len()
     }
 
-    fn compute_clobbers(&self, regalloc: &regalloc2::Output) -> Vec<Writable<RealReg>> {
+    fn compute_clobbers(&self, regalloc: &regalloc2::Output) -> (Vec<Writable<RealReg>>, bool) {
```

Comment:
Let's call this `compute_clobbers_and_leaf` for clarity, now that it returns both.